rules/java_rules.build_defs

```diff
@@ -45,7 +48,7 @@ def java_library(name:str, srcs:list=None, src_dir:str=None, resources:list=[],
         else:
             javac_flags = CONFIG.JAVAC_TEST_FLAGS if test_only else CONFIG.JAVAC_FLAGS
 
-        javac_flags = '-g -encoding utf8 ' + javac_flags
+        javac_flags = '-h _jni -g -encoding utf8 ' + javac_flags
```

Comment:
There doesn't seem to be any significant overhead in invoking the compiler with `-h` even when the JNI headers aren't needed (at least not compared with the time it takes to generate the class files).